packages/app/src/screens/TabsNavigation/screens/Map/config.ts

```diff
@@ -42,13 +42,13 @@ const markers: Marker[] = [
     navigation: '54.179019,15.5368783',
     type: 'outline',
   },
-  // {
-  //   title: 'Promenada',
-  //   address: '???, 78-100 Kołobrzeg',
-  //   coordinates: [, ],
-  //   navigation: '',
-  //   type: 'umbrella',
-  // },
+  {
+    title: 'Promenada',
+    address: 'Plac przy Molo, bulwar Jana Szymańskiego',
+    coordinates: [54.186373, 15.561874],
```

Comment:
```suggestion
    coordinates: [15.561874, 54.186373],
```
MapBox is using lat/lng coordinates in reverse

Reply:
right, thanks!